source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
It's not acceptable for react-virtualized users to have to call `forceUpdate` to get correct layout behavior. The server-side rendering edge-case we're talking about can be detected and handled by `Grid` automatically by checking for `this._scrollbarSize == null`. At that point your tests- (and more important, users using `Grid`- won't have to call `forceUpdate` themselves.

Does this make better sense?